pom.xml

```
@@ -8,39 +8,42 @@
     <module>service</module>
   </modules>
 
-  <groupId>mx.klar.sample</groupId>
+  <groupId>mx.klar.incode</groupId>
   <artifactId>parent</artifactId>
   <version>1.0.0</version>
   <packaging>pom</packaging>
-  <name>_sample_ service - parent</name>
-  <description>_sample_ project parent</description>
+  <name>incodeService - parent</name>
```

big NIT: 
```suggestion
  <name>Incode service - parent</name>
```
the same in other places 😬